doc/man/kdb-export.1

```diff
@@ -69,25 +73,25 @@ To view your full key database in Elektra’s \fBstorage\fR format:
 .br
 \fBkdb export /\fR
 .
-.P
+.br
 To backup your full key database in Elektra’s \fBstorage\fR format to a file called \fBfull\-backup\.ecf\fR:
 .
 .br
 \fBkdb export / > full\-backup\.ecf\fR
 .
-.P
+.br
 To view a keyset stored in \fBuser/keyset\fR in the XML format:
 .
 .br
 \fBkdb export user/keyset xmltool\fR
 .
-.P
```

Comment:
Seems like it is not completely the same for ronn?

Reply:
I updated the man pages again. The only place where the update did anything, besides adding additional `.br` commands,  is now `kdb-rm.md`.